omod/src/main/webapp/resources/htmlforms/Covid19-Dischargeform.xml

```diff
@@ -0,0 +1,777 @@
+                                            <div class="row">
+                                                <div class="col-md-3 anti-biotics">
+                                                    <label>Antibiotics</label>
+                                                    <obs conceptId="166421" answerConceptIds="90003,90004" answerLabels="Yes, No"
+                                                         style="radio" class="horizontal" id="malaria-treatment"/>
+                                                </div>
+                                                <div class="col-md-3">
+                                                    <label>Specify Antibiotics</label>
+                                                    <obs id="specify-other-anti-biotics" conceptId="166344" />
+                                                </div>
+                                                <div class="col-md-3">
+                                                    <label>Novel/Experimental Therapeutic agent</label>
+                                                    <obs conceptId="166361" answerConceptIds="90003,90004, 90001" answerLabels="Yes, No, Unknown"
+                                                         style="radio" class="horizontal" id="novel-experimental-therapeutic-agent"/>
+                                                </div>
+                                                <div class="col-md-3">
+                                                    <label>Specify Novel/Experimental Therapeutic agent</label>
+                                                    <obs id="specify-novel-experimental-therapeutic-agent" conceptId="166344" />
+                                                </div>
+                                            </div>
+                                    </div>
+                                </div>
+                                <div class="card">
+                                    <div class="card-body">
+                                        <div class="row">
+                                            <div class="col-md-2">
+                                                <label>Hydroxychloroquine</label>
+                                                <obs conceptId="166348" answerConceptIds="90003,90004" answerLabels="Yes, No"
+                                                     style="radio" class="horizontal" id="hydroxychloroquine"/>
+                                            </div>
+
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="card">
+                                    <div class="card-body">
+                                        <div class="row">
+                                            <div class="col-md-2">
+                                                <label>Antiviral Given</label>
```

Comment:
Am going to create a new concept for antiviral given (to be used here) to distinguish it from antiviral (166351)

Reply:
waiting for the concept id